graphql/statistics/fields.ts

```diff
@@ -721,4 +817,40 @@ export class StatisticsResolver {
 
         return await prisma.secret.count({ where: { lastUsed: { gte: beginingOfTheDay } } });
     }
+
+    @FieldResolver(() => Int)
+    @Authorized(Role.ADMIN)
+    async helpersWithoutScreening(@Root() statistics: Statistics) {
+        return (
+            await prisma.$queryRaw`
+            SELECT COUNT(*)::int AS value
+            FROM student
+            LEFT JOIN instructor_screening ON student.id = instructor_screening."studentId"
```

Comment:
this also needs to take screenings into account, not just instructor_screenings?

Reply:
Oh this endpoint should actually be called tutorsWithoutMatch, as it's for the tutoring statistics page. Good catch!

Reply:
Before someone was screened, we do not know whether they will be tutors or instructors?

Reply:
The query should query for screenings, not instructor_screenings.

Hmm, is there no way to know how many wannabe-tutors have been invited for screening? 
What about `isStudent=true and (not exists screening where screening.studentId = student.id)`?

Reply:
that will be 0 for now

Reply:
Oh I see. I thought a student called the `meBecomeTutor` endpoint by themselves, but that seem to not be the case in the user app.